src/app/common/loading-modal/loading-modal.component.scss

```diff
@@ -1,9 +1,13 @@
 .snackbar {
   display: flex;
-  font-size: 32px;
+  font-size: 16px;
   align-items: center;
 
   > mat-spinner {
-    margin-right: 50px;
+    margin-right: 15px;
   }
 }
+
+::ng-deep .custom-snackbar-class {
```

Comment:
Probably better will be to create in `/assets/styles/` directory `components` and there file `snackbar.scss`. Then create a class there like `app-snackbar` or something similar. Those styles are not excatly styles of this component but we are overriding styles of angular material component so this should be global css class.